storage/innobase/srv/srv0start.cc

```diff
@@ -2221,10 +2221,20 @@ dberr_t srv_start(bool create_new_db IF_XB(, lsn_t to_lsn)) {
 
       ut_ad(buf_pool_pending_io_reads_count() == 0);
 
+#ifdef XTRABACKUP
+      /* we have to recreate always in PXB because the ib_redo0 file
```

Comment:
its an existing mistake, can you add "we have to recreate redo log file always during PXB prepare because the ..."

Reply:
Done